lib/plugins/chat.js

```diff
@@ -131,10 +131,19 @@ function inject (bot, options) {
       const { profileKeys } = bot.players[sender]
       if (profileKeys) verified = bot._client.verifyMessage(profileKeys.publicKey, packet)
     }
-    const parameters = {
-      sender: JSON.parse(packet.senderName),
-      content: JSON.parse(message)
+    let parameters = {};
```

Comment:
This code is messy.

Consider doing something like:
```js
const parameters = {
    sender: JSON.parse(packet.networkName ? packet.networkName: packet.senderName),
    content: JSON.parse(packet.networkName ? { "text": packet.plainMessage } : message)
  }

Reply:
I think the code in the PR is better actually. Makes more sense to branch on this once.
Could maybe be improved by using support feature though

Reply:
This commit in general is just "here's the minimum diff to let us verify the minecraft-data changes and make it easier to continue development"

Reply:
how long would it take  for this bot to update to 1.19.2